EIPS/eip-draft_security_considerations.md

```diff
@@ -14,27 +14,14 @@ requires: 1
 ## Simple Summary
 <!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
 
-This document describes an improvement to the Ethereum Improvment Proposal (EIP) system as defined in EIP-1 to explicitly embed security into the EIP process by visibly documenting security related information and considerations in a dedicated "Security Considerations" section of the EIP, adapted from the Request for Comments (RFC) system ([rfc7322 - section 4.8.5](https://tools.ietf.org/html/rfc7322#section-4.8.5)).
-
+Following the Constantinople fork postponement, all EIPs MUST include a "Security Considerations" section. Requirements include discussion of security considerations in the design, indicating which aspects of the protocol may be affected (networking, virtual machine, etc.), listing risks and how they have been mitigated. This proposal is adapted from the IETF's Request for Comments (RFC) system ([rfc7322 - section 4.8.5](https://tools.ietf.org/html/rfc7322#section-4.8.5)).
 
 ## Abstract
 <!--A short (~200 word) description of the technical issue being addressed.-->
 
-The EIP system defines a minimum set of information and criteria to be included in a proposal. Security discussions take place but important considerations are not always documented with the EIP. Furthermore, security related information and design decisions might blend into technical documentation, lacking visibility and not being easily available for the security community posing a greater threat for changing security aspects to not be considered in later phases of the EIP (e.g. implementation phase). Given the impact of proposed changes to the ethereum protocol (e.g. [EIP-1057 - PoW change Ethhash to ProgPoW](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1057.md)) and a recently surfacing security vulnerability introduced with [EIP-1283](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1283.md) that was part of the initial Constantinople hard-fork and following discussions about maturing the EIP system to better address security in the change management process we propose to pro-actively encourage a security discussion by adding a "Security Considerations" section to the minimum set of information provided with a EIP. It is important to document security considerations as part of the EIP to better manage and understand the risks introduced by a EIP as well as to provide as a basis for exploring and assessing the potential impact of the proposed change. 
-
-The goals of requiring a mandatory "Security Considerations" section are as follows:
```

Comment:
No content was actually removed, but I moved these two lists into the sections below.

Reply:
👍